web-dev/src/components/RegistrationPage/index.js

```diff
@@ -1,9 +1,32 @@
-import React from "react";
+import React, {useRef}from "react";
+import * as service from '../../services/auth-service'
 import "../LoginPage/login-page.css"
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import LoginBackground from "../LoginPage/login-background.jpg";
+import {useAuth} from "../../contexts/auth-context";
 
 const RegistrationPage = () => {
+    const firstNameRef = useRef();
+    const lastNameRef = useRef();
+    const userNameRef = useRef();
+    const emailRef = useRef();
+    const passwordRef = useRef();
+    const navigate = useNavigate();
+    const {signUp} = useAuth();
+    const handleSignup = async () => {
+        try {
```

Comment:
Please add comments to the try block in order to make it more clear what the logic is. Thanks!